src/ProgressOnderwijsUtils/Pocos/BulkInsertTarget.cs

```diff
@@ -33,10 +36,13 @@ public static BulkInsertTarget FromCompleteSetOfColumns(string tableName, IDbCol
         => new(tableName, columns.ArraySelect(ColumnDefinition.FromDbColumnMetaData));
 
     public BulkInsertTarget With(BulkCopyFieldMappingMode mode)
-        => new(TableName, Columns, mode, Options);
+        => new(TableName, Columns, mode, Options, ReadOnlyTarget);
 
     public BulkInsertTarget With(SqlBulkCopyOptions options)
-        => new(TableName, Columns, Mode, options);
+        => new(TableName, Columns, Mode, options, ReadOnlyTarget);
+
+    public BulkInsertTarget With(ReadOnlyTargetMode mode)
+        => new(TableName, Columns, Mode, Options, mode);
```

Comment:
We zouden kunnen overwegen deze "settings"-achtige class een record te maken, en dan kunnen we de native `with` syntax gebruiken.  Echter, dat zou wel een breaking change zijn en een klein beetje ook aan de consumer-side refactoring nodig maken, dus misschien is dat zo prima.